lib/maths/unittest/CBoostedTreeTest.cc

```diff
@@ -268,41 +289,43 @@ BOOST_AUTO_TEST_CASE(testPiecewiseConstant) {
         };
     };
 
-    TDoubleVecVec modelBias;
-    TDoubleVecVec modelRSquared;
-
     test::CRandomNumbers rng;
     double noiseVariance{0.2};
     std::size_t trainRows{1000};
```

Comment:
Maybe cut this down a bit if you're testing all three loss functions:
```suggestion
    std::size_t trainRows{500};
```

Reply:
Fixed